src/main/java/Client/ConsoleClientLauncher.java

```diff
@@ -28,17 +27,25 @@
     private static final Logger logger = LoggerFactory.getLogger(ConsoleClientLauncher.class);
     public static Future<Void> clientListening;
 
+    static {
+        socket = null;
+        client = null;
+        menu = null;
+    }
+
```

Comment:
You are absolutely right, I was insecure about supporting more clients at the same time, so i was trying to find a proper solution. Because if there is only one Launcher with static methods, multiclients are not possible or am i mixing things up in this case?